.github/workflows/build-and-test.yml

```diff
@@ -12,9 +12,20 @@ jobs:
           - name: Ubuntu 20.04
             os: ubuntu-20.04
             install_dir: ~/libKeyFinder
+            cmake_extras: -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Comment:
This is common to all matrix conditions. How about putting it directly in the build step unconditionally?

Reply:
It isn't, the Visual Studio generator is multi-config and needs `--config RelWithDebInfo` during the build step instead.